src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
```

Comment:
Why true?
Put here valid data
```suggestion
    switch (action.type) {
```

Reply:
I recommend you take a look at this [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)